### PR TITLE
Bugfix abc 984 image magnifier option

### PR DESCRIPTION
**Pull request by Jusolijus17**
### Fixes
**Image**
- Fixing small issues

### CODE REVIEW

**Review by nawel-aa — CHANGES_REQUESTED** [review]
One of the tests is failing. Can you check if it should still pass, or update it otherwise?

File: src/modules/base/image/image.js [hunk 1]
```diff
@@ -347,15 +347,19 @@ export default class Image extends LightningElement {
...
-        )
-            ? normalizedAttributes.zoomFactor
-            : DEFAULT_ZOOM_FACTOR;
+        if (
+            normalizedAttributes.zoomFactor &&
+            !isNaN(normalizedAttributes.zoomFactor)
+        ) {
+            this._magnifierAttributes.zoomFactor =
+                normalizedAttributes.zoomFactor;
+        }
```
Comment (nawel-aa) [inline src/modules/base/image/image.js:362]: Here it is impossible to reset the `zoomFactor` and the `smoothMove` keys. If magnifier attributes were passed, and then removed, their value would remain.

File: src/modules/base/image/image.css [hunk 2]
```diff
@@ -55,6 +55,7 @@
     border-width: var(--avonni-image-magnifier-sizing-border, 1px);
     border-style: var(--avonni-image-magnifier-styling-border, solid);
     border-color: var(--avonni-image-magnifier-color-border, #ccc);
+    border-radius: var(--avonni-image-magnifier-radius-border, 0);
```
Comment (nawel-aa) [inline src/modules/base/image/image.css:58]: Good idea! But if you add a styling hook, you need to document it in `image.jsdoc.js` and [Confluence](https://avonnicreator.atlassian.net/wiki/spaces/ABC/pages/363593765/Image). 😊

**Review by nawel-aa — CHANGES_REQUESTED** [review]

File: src/modules/base/image/image.css (hunk 2 above)
Comment (Jusolijus17) [reply src/modules/base/image/image.css:58]: Yes I remembered right after requesting your review🤦‍♂️

**Review by nawel-aa — APPROVED** [review]

**Pull request merged**